mesa/experimental/cell_space/voronoi.py

```diff
@@ -0,0 +1,264 @@
+from collections.abc import Sequence
+from itertools import combinations
+from random import Random
+
+import numpy as np
+
+from mesa.experimental.cell_space.cell import Cell
+from mesa.experimental.cell_space.discrete_space import DiscreteSpace
+
+
+class Delaunay:
```

Comment:
Should we put this in a separate file? @rht?

Reply:
In voronoi.py should be fine. We can split it once it is used in more than one context. The docstring should clearly indicate that this is a helper class, and not a space implementation it itself. I'd rename it to `_Delaunay` for now so that we can merge this PR sooner. We can discuss about other use cases later.